packages/core/core/src/services/document-service/transform/relations/transform/data-ids.ts

```diff
@@ -178,10 +171,9 @@ const transformDataIdsVisitor = (
       // Find relational attributes, and return the document ids
       if (attribute.type === 'relation') {
         const target = attribute.target as Common.UID.Schema | undefined;
+        // If uid doesn't target a content type, ignore any transformation
         // TODO: Handle polymorphic relations
-        if (!target) return;
-        // TODO: V5 remove excluded fields and use { id: } syntax for those relations
-        if (EXCLUDED_FIELDS.includes(key)) return;
+        if (!target || !isContentType(target)) return;
```

Comment:
🤔 which case are you thinking about ?

Reply:
a user creating a relation to one of our internal models that are not content types